docker.go

```diff
@@ -45,9 +48,10 @@ func (docker *dockerConfig) call() int {
 	// Change the default log level for terragrunt
 	const logLevelArg = "--terragrunt-logging-level"
 	if !util.ListContainsElement(command, logLevelArg) && filepath.Base(config.EntryPoint) == "terragrunt" {
-		if config.LogLevel == "6" || strings.ToLower(config.LogLevel) == "full" {
-			config.LogLevel = "debug"
-			config.Environment["TF_LOG"] = "DEBUG"
+		level, _ := strconv.Atoi(config.LogLevel)
```

Comment:
Should we also set a more proper default if LogLevel is wrong, like "abc"?  With the current code that will silently set the level to 0 (panic) so almost no logs...  Is that what we want?  Should we default to something more verbose to like "info" in this case?

Reply:
You are right that we should add a warning, but the effect of a wrong level name here is just to ignore it. The goal is just to trap special cases where the logging level is expressed as a number and is greater than 6 or set to full. If it is not the case, the logging level will simply be passed to terragrunt and should be managed there. In fact, there is actually no real logging implemented in tgf.

I am working on another PR where I implement a more intelligent logging directly in tgf.